Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/submissions/use-submissions-details.tsx

```diff
@@ -204,10 +192,13 @@ const SubmissionsDetailsProvider = ({ children }: ISubmissionsDetailsProviderPro
     const getDetails = useCallback(
         async (submissionId: number) => {
             if (isNil(submissionId) || Number.isNaN(submissionId)) {
-                return;
+                // return;
             }
```

Comment:
There is a sub issue for these changes, because there will be also code that it is not used anymore, https://github.com/SoftUni-Internal/exam-systems-issues/issues/937

Reply:
Same here.